### PR TITLE
añadidio task06

### PR DESCRIPTION
**Pull request by josuesalas25**

### CODE REVIEW

File: Assignment4/josuesalas25-170075/task07.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+"""**TASK 7.1: List all subclasses of "Person" with RDFLib and SPARQL**"""
+
+from rdflib.plugins.sparql import prepareQuery
+
+ns = Namespace("http://somewhere#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+
+print("RDFLib --> ")
+for s, p, o in g:
+  print(s,p,o)
```
Comment (dgarijo) [inline Assignment4/josuesalas25-170075/task07.py:33]: This is printing all the triples in the graph, not just the Persons

**Comment by dgarijo** [comment]
@josuesalas25,
this is almost correct (see comment)

**Comment by josuesalas25** [comment]
Ya he actualizado el fichero task07.py

**Comment by rgcmme** [comment]
* There is already a group06. Please, select another number.
* The self-assessment must be filled with "x".

These changes must be corrected before merging the pull request.

**Comment by josuesalas25** [comment]
Creo que ya están todos los cambios realizados

**Comment by rgcmme** [comment]
Thanks, @dgarijo once assignment 4 is correct, this pull request can be merged.

**Comment by josuesalas25** [comment]
Voy a cambiar y poner cada entrega pendiente por aprobar con un pull request diferente

**Pull request closed without merging**